turn on dim order in cadence test

Gasoonjia opened the pull request:
Summary: as title

Reviewed By: digantdesai

Differential Revision: D68246404

Gasoonjia commented:
@pytorchbot label "topic: not user facing"

mcremon-meta reviewed (approved):
Looks like lint error is fairly straightforward, sorry about that! And I guess I did have extra parentheses on that `assert` too

Gasoonjia commented:
> Looks like lint error is fairly straightforward, sorry about that! And I guess I did have extra parentheses on that `assert` too

Thanks @mcremon-meta for help! Will update first then land it

Gasoonjia committed:
turn on dim order in cadence test (#7756)

Summary:


This diff turns on dim order in cadence test. Also we get around `to_copy` operator in verifier to keep the verifier check enable.

Reviewed By: mcremon-meta

Differential Revision: D68246404
